test/test_crud_v1.py

```diff
@@ -158,12 +159,21 @@ def run_operation(collection, test):
 
 def create_test(scenario_def, test):
     def run_scenario(self):
-        # Load data.
-        assert scenario_def['data'], "tests must have non-empty data"
+        # Cleanup state and load data (if provided).
         drop_collections(self.db)
-        self.db.test.insert_many(scenario_def['data'])
-
-        result = run_operation(self.db.test, test)
+        data = scenario_def.get('data')
+        if data:
+            self.db.test.insert_many(scenario_def['data'])
+
+        # Run operations and check results or errors.
+        expected_result = test.get('outcome', {}).get('result')
```

Comment:
I think this is supposed to be `test.get('result')`. I don't see outcome.result in the new format.